apps/tangle-dapp/data/payouts/useNominationsUnclaimedRewards.ts

```diff
@@ -69,6 +83,9 @@ export default function useNominationsUnclaimedRewards() {
         // If the reward has claimed, do nothing
         if (hasClaimed) return;
 
+        // If the era is not in the range, do nothing
+        if (!erasRange.includes(era)) return;
```

Comment:
Use number comparison here to check the in-range era instead of array include `(startEra <= era && era <= endEra)`.